.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!--
+If you want to report a bug, we added some points below you can fill out. If you want to request a feature, feel free to remove all the irrelevant text. In addition to this [issue tracker](https://github.com/diesel-rs/diesel/issues), you can also talk to Diesel maintainers and users on [Gitter](https://gitter.im/diesel-rs/diesel).
+-->
+
+## Setup
+
+### Versions
+
+- **Rust:**
+- **Diesel:**
+- **Database:**
+- **Feature Crates:**
```

Comment:
If you're using Chrono as a diesel feature, what version are you using?

I thought it'd be good to add the versions of any diesel "plug-ins" they may be using.